File: 3393/3393.py
class Solution:
    def count_paths_with_the_given_xor_value(self, root: Optional[TreeNode]) -> int:
        if not root:
            return 0
        return 1 + max(self.count_paths_with_the_given_xor_value(root.left), self.count_paths_with_the_given_xor_value(root.right))

File: 3393/3393.go
func countPathsWithTheGivenXorValue(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := countPathsWithTheGivenXorValue(root.Left)
	r := countPathsWithTheGivenXorValue(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}